package.json

```diff
@@ -84,7 +84,7 @@
     "@types/bluebird": "3.5.21",
     "@types/chokidar": "^1.7.5",
     "@types/common-tags": "1.4.0",
-    "@types/dockerode": "2.5.5",
+    "@types/dockerode": "2.5.6",
```

Comment:
`2.5.6` adds the `dockerode.DockerVersion` type. `@types/dockerode` is now at `2.5.20`, but anything higher than `2.5.6` breaks compatibility with several other repos (including `resin-multibuild`) which are pinned to `2.5.5` in their `package.json`.

Reply:
Ugh, we should really look at upgrading the other modules, I've gotten a few patches into `@types/dockerode` recently which have added a lot.